src/sql/_testing.py

```diff
@@ -178,6 +178,18 @@ def get_tmp_dir():
             "service_name": "XEPDB1",
         },
     },
+    "redshift": {
+        "drivername": "redshift+redshift_connector",
+        "username": os.getenv("REDSHIFT_USERNAME"),
+        "password": os.getenv("REDSHIFT_PASSWORD"),
+        # database/schema
```

Comment:
do we need it?

Reply:
yes, they're stored as github secrets. I'm using `get` here so it doesn't break when loading the module. I have another function that throws an error if they're missing